.travis.yml

```diff
@@ -22,7 +22,7 @@ before_install:
   - sudo rm -f /etc/boto.cfg
 env:
   global:
-    - CENTAUR_BRANCH=develop
+    - CENTAUR_BRANCH=rm_monitoring
```

Comment:
Centaur JES is failing because it's expecting the script to write a `monitor.log` file. As there isn't one, the JES/PAPI delocalizer is crashing.